client/components/status/status.js

```
@@ -90,160 +90,188 @@ export default class Status extends React.Component {
 
-      // stop here if token is invalid or component unmounted
-      if (isValid === false || !this.isComponentMounted) {
-        setLoading(false);
-        return;
-      }
+    if (isValid === false || !this.isComponentMounted) {
+      setLoading(false);
+      return;
+    }
+    const {setLoading} = this.context;
+    let {userData} = this.props;
```

ESLint

[error] 'userData' is never reassigned. Use 'const' instead.

(prefer-const)